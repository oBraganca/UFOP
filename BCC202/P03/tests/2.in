5
I pedro
125435 palio 4 2018 pedro 213
425435 gol 5 2012 carlos 212
987654 palio 5 2019 pedro 213
785452 fox 5 2021 carlos 212
813429 ferrari 2 2023 pedro 213