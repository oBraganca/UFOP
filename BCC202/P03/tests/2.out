ferrari [2023|2|813429]
palio [2019|5|987654]
palio [2018|4|125435]
